Documentation/Upgrade/rook-upgrade.md

```diff
@@ -43,27 +44,11 @@ those releases.
     official releases. Builds from the master branch can have functionality changed or removed at any
     time without compatibility support and without prior notice.
 
-## Breaking changes in v1.9
+## Breaking changes in v1.10
 
-* Helm charts now define default resource requests and limits for Rook Pods. If you use Helm,
-  ensure you have defined an override for these in your `values.yaml` if you don't wish to use the
-  recommended defaults. Setting resource requests and limits could mean that Kubernetes will not
-  allow Pods to be scheduled in some cases. If sufficient resources are not available, you can
-  reduce or remove the requests and limits.
+* Support for Ceph Octopus (15.2.x) was removed. If you are running v15 you must upgrade to Ceph
+  Pacific (v16) or Quincy (v17) before upgrading to Rook v1.10
 
```

Comment:
We may have a more strict csi upgrade policy depending on [this conversation](https://github.com/rook/rook/pull/10759#discussion_r953014295).

Reply:
Added a note about CSI v3.6.0 being the min supported version